libraries/testkit/cluster.py

```diff
@@ -192,7 +173,10 @@ def validate_cbgt_pindex_distribution_retry(self):
 
         return False
 
-    def validate_cbgt_pindex_distribution(self):
+    def validate_cbgt_pindex_distribution(self, num_running_sg_accels):
+
+        if num_running_sg_accels < 1:
+            raise keywords.exceptions.ClusterError("Need at least one sg_accel running to verify pindexes")
```

Comment:
Not necessarily needed for this PR, but it looks like we aren't validating that PIndexes are balanced across the nodes.  e.g. after adding a second node, could end up w/ 64 PIndexes on one node, and 0 on the other, and this would still pass.